resolwe/flow/executors/docker/__init__.py

```diff
@@ -30,15 +36,56 @@ def start(self):
             'container_image': self.requirements.get('image', settings.FLOW_EXECUTOR['CONTAINER_IMAGE']),
         }
 
+        # Set resource limits.
+        limits = []
+        if 'cores' in self.resources:
+            # Each core is equivalent to 1024 CPU shares. The default for Docker containers
+            # is 1024 shares (we don't need to explicitly set that).
+            limits.append('--cpu-shares={}'.format(int(self.resources['cores']) * 1024))
```

Comment:
There are some processes that supports setting number of cores as input parameter (i.e. [cuffquant](https://github.com/genialis/resolwe-bio/blob/ed1c342c8517bb8625e381b2e299ded790105706/resolwe_bio/processes/expression/cuffquant.yml#L56). ). How will this be handled?